considered/rfc-oop-fields.rst

```diff
@@ -53,6 +53,57 @@ For example:
 
 Visibilty rules of these fields are following the usual package-visibilty rules.
 
+It is possible to declare empty public or private views either with an empty `record end record` section, or the shorthands `null record` syntax.
+
+For example, all of the following type declarations are legal:
+
+.. code-block:: ada
+
+   package Legal is
+      type R1 is null record with private;  -- This syntax is supported for declaring an empty public view
+
+      type R2 is record
+         Pub : Integer;
+      end record
+      with private;
+
+      type R3 is record
+      end record with private;  -- This record type has both public and private views empty
```

Comment:
To be consistent with Ada, this should be: 
type R3 is record
   null;
end record with private;

Reply:
Fixed!